docs/en/UM_Authentication.md

```diff
@@ -9,30 +9,40 @@ layout: page
 
 To maintain the security of your applications and users, all requests to Adobe I/O APIs must be authenticated and authorized using standards such as OAuth and JSON Web Tokens (JWT). Before you can access the User Management API, you will need to obtain access credentials by creating a new Integration in the Adobe I/O Console or Admin Console.
 
-![Diagram of Authentication workflow](AuthWorkflow.png)
 
 ## One-time Setup
 
-To obtain the credentials you need to access the User Management service, create a **Service Account Integration** using the [Adobe I/O Console](https://console.adobe.io/).
+To obtain the credentials you need to access the User Management service, create a **Project** using the [Adobe Developer Console](https://developer.adobe.com/).
 
-* A member of the organization with admin rights can create the Integration. You can use the Admin Console to grant administrative privilege to users.
-* You will need to create or purchase a digital signing certification, and use it to sign a JSON Web Token (JWT), which you will use to authenticate your requests.
+* Only members of the organization with SYSTEM ADMIN role can create the Integration for UMAPI. You can use the Admin Console to grant administrative privilege to users.
+* For JWT workflows, you will need a keypair (public certificate and private key). One can be generated from developer portal, during the Project creation phase
 * Your integration provides an API key that uniquely identifies your client, and other credentials that you need to access the UM API.
 
-For complete details of the entire integration process, see [Service Account Authentication](https://developer.adobe.com/developer-console/docs/guides/authentication/ServiceAccountIntegration/).
```

Comment:
It would be good to try and keep this logic. We would prefer that most of the documentation referred to the developer.adobe.com rather than the risk of the UMAPI content going stale.